iset.mm

```diff
@@ -117163,6 +117163,19 @@ Infinite sums (cont.)
     ALZMNUQHUSOJZDEZVAURUQUSPCHRCVDVAUAUQUSVBUBUCUSHUDUEUQVCADUQAHAUFZUQUGHGSTU
     QUHUKUIUJULUMUQAPCURGSTVEAUNUOUP $.
 
+  $( The exponential function of a real number is a positive real.
```

Comment:
The exponential of a...

Reply:
Changed in both set.mm and iset.mm

Reply:
But shouldn't "the exponential function of" be replaced by "the exponential of" also in the following theorems (~efgt0 etc.)?

Reply:
Probably "MM> set width 9999" followed by a search for "exponential function of" will allow to detect most cases.